test/functional/feature_block.py

```diff
@@ -502,7 +502,7 @@ def run_test(self):
         # Make sure we didn't accidentally make too big a block. Note that the
         # size of the block has non-determinism due to the ECDSA signature in
         # the first transaction.
-        while (len(b39.serialize()) >= MAX_BLOCK_BASE_SIZE):
+        while (b39.get_weight() >= MAX_BLOCK_WEIGHT):
```

Comment:
e26ad3b39cf7afcf5588bb55ceaf3c3902f7b5d3: Can remove `()`

Reply:
Thanks, done. Also identified another pair of unneeded parantheses in an if condition to remove.